code/oled_block/README.md

```diff
@@ -0,0 +1,11 @@
+# OLED_BLOCK
+
+### Get this bad boy working:  
+Just follow [this](https://www.raspberrypi-spy.co.uk/2018/04/i2c-oled-display-module-with-raspberry-pi/) tutorial.
+
+### How it works?
+Program should have access to oled_buffer.txt file, you can specify the path in the code.  
```

Comment:
if it's configurable, changes in code shouldn't be the default way of interaction with an user, please use [argparse](https://docs.python.org/3/library/argparse.html) or something similar

Reply:
default values for these arguments are also required

Reply:
added config file :smile_cat: